src/domains/sign/storage.tf

```diff
@@ -13,10 +13,39 @@ module "io_sign_storage" {
   allow_blob_public_access                     = false
   blob_properties_delete_retention_policy_days = var.storage.delete_retention_policy_days
   tags                                         = var.tags
+  network_rules = {
+    default_action = "Deny"
+    ip_rules       = []
+    bypass = [
+      "Logging",
+      "Metrics",
+      "AzureServices",
+    ]
+    virtual_network_subnet_ids = [
+      module.io_sign_snet.id
+    ]
+  }
 }
 
-resource "azurerm_storage_container" "documents" {
-  name                  = "documents"
+resource "azurerm_storage_container" "uploaded_documents" {
+  name                  = "uploaded-documents"
   storage_account_name  = module.io_sign_storage.name
   container_access_type = "private"
-}
\ No newline at end of file
+}
+
+resource "azurerm_storage_container" "validated_documents" {
+  name                  = "validated-documents"
+  storage_account_name  = module.io_sign_storage.name
+  container_access_type = "private"
+}
+
+resource "azurerm_storage_container" "signed_documents" {
+  name                  = "signed-documents"
+  storage_account_name  = module.io_sign_storage.name
+  container_access_type = "private"
+}
+
+resource "azurerm_storage_queue" "waiting_message" {
```

Comment:
this queue is no longer used, we can remove it

Reply:
If I remove the queue, how would the two function exchange signature requests?

Reply:
@lucacavallaro I suggest to keep one single queue. What do you think?